gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy

```diff
@@ -305,6 +306,18 @@ interface ClouddriverService {
   @GET('/tags')
   List<Map> listEntityTags(@QueryMap Map allParameters)
```

Comment:
Should we remove this method? Or is there some value in having both exposed?

Reply:
Hmm. Let me remove that! I don't think it's needed.

Reply:
Aside from that, I think this is good! It may be worth adding `@ApiParam` annotations to explain things like `entityId` can be comma-separated, but what you have here is a big improvement so let's ship it and consider adding extra complexity later.